contrib/mesos/pkg/scheduler/components/schedulerloop/schedulerloop.go

```diff
@@ -0,0 +1,107 @@
+/*
+Copyright 2015 The Kubernetes Authors All rights reserved.
+
+Licensed under the Apache License, Version 2.0 (the "License");
+you may not use this file except in compliance with the License.
+You may obtain a copy of the License at
+
+    http://www.apache.org/licenses/LICENSE-2.0
+
+Unless required by applicable law or agreed to in writing, software
+distributed under the License is distributed on an "AS IS" BASIS,
+WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+See the License for the specific language governing permissions and
+limitations under the License.
+*/
+
+package schedulerloop
+
+import (
+	"time"
+
+	log "github.com/golang/glog"
+	"k8s.io/kubernetes/contrib/mesos/pkg/runtime"
+	"k8s.io/kubernetes/contrib/mesos/pkg/scheduler/components/algorithm"
+	"k8s.io/kubernetes/contrib/mesos/pkg/scheduler/components/binder"
+	"k8s.io/kubernetes/pkg/api"
+	"k8s.io/kubernetes/pkg/client/record"
+	client "k8s.io/kubernetes/pkg/client/unversioned"
+)
+
+const (
+	recoveryDelay = 100 * time.Millisecond // delay after scheduler plugin crashes, before we resume scheduling
+
+	FailedScheduling = "FailedScheduling"
+	Scheduled        = "Scheduled"
+)
+
+type SchedulerLoop interface {
```

Comment:
I would like to keep the pattern with `Run()` it's used everywhere else in the code where background go routines are launched.